frontends/dotty/src/main/scala/stainless/frontends/dotc/ASTExtractors.scala

```diff
@@ -1401,6 +1406,30 @@ trait ASTExtractors {
       }
     }
 
+    // object ExCellApply {
+    //   def unapply(tree: tpd.Apply): Option[(tpd.Tree, tpd.Tree)] = tree match {
+    //     case Apply(TypeApply(ExSymbol("stainless", "lang", "Cell$", "apply"), List(tpt)), value :: Nil)  => Some(( value, tpt))
+    //     case _ => None
+    //   }
+    // }
```

Comment:
@Mario-bucev (see above question) if yes, how should I do it here?

Reply:
Not necessary :)

Reply:
Perfect :)